src/Tribe/Views/V2/Views/Month_View.php

```diff
@@ -346,12 +346,13 @@ static function ( $event ) use ( $date_object ) {
 			);
 
 			$start_of_week = get_option( 'start_of_week', 0 );
+			$is_start_of_week = (int)$start_of_week === (int)$date_object->format( 'w' );
 
 			$day_url = tribe_events_get_url( [ 'eventDisplay' => 'day', 'eventDate' => $day_date ] );
 
-			$day_data = [
+			$day_data         = [
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Equals sign not aligned correctly; expected 1 space but found 9 spaces

<sub>hash: 0833adb88f44bffa92ca666345ea40e2</sub>